addons/ui/org.openhab.ui.cometvisu/META-INF/MANIFEST.MF

```diff
@@ -52,5 +52,4 @@ Bundle-ClassPath: .,
  lib/sqlite-jdbc-3.8.7.jar
 Export-Package: org.openhab.ui.cometvisu.php
 Bundle-Activator: org.openhab.ui.cometvisu.internal.CvActivator
- 
- 
+Bundle-ActivationPolicy: lazy
```

Comment:
No, after the discussion we had recently I agree to keep the "lazy" until somebody fixed the IDE.